ui_test/README.md

```diff
@@ -48,3 +48,6 @@ their command specifies, or the test will fail without even being run.
 
 * `ignore-*` and `only-*` opereate solely on the triple, instead of supporting things like `macos`
 * only `//~` comments can be individualized per revision
+* no "silent mode" (one `.` per test, instead of printing names)
```

Comment:
I think you fixed this one :)  (Well, the PR hasn't landed yet, but will soon. So no need to add it here I think.)